Final_Project_Billinis-AR.qmd

```diff
@@ -0,0 +1,132 @@
+ggsave("Oct.png")
+
+```
+
+2.  Greek Ships Sunk Prior to German Invasion of Greece: April 6, 1941
+
+```{r}
+library(ggplot2)
+Grcategory <- Pre_April %>% group_by(Event) %>% summarize(count = n())
+ggplot(Grcategory, aes(x = Event, y = count)) + geom_col() + ggtitle("Ships Sunk Pre April 6, 1941")
+ggsave("Apr.png")
+```
+
+3.  American Ships Sunk prior to Pearl Harbor (December 7, 1941)
+
+```{r}
+Amships <- read.csv("USShips_Pre12_7.csv")
+Amcategory <- Amships %>% group_by(Flag) %>% summarize(count = n()) 
+ggplot(Amcategory, aes(x = Flag, y = count)) + geom_col() + ggtitle("US Controlled Ships Sunk, Pre December 7, 1941")
+ggsave("USships.png")
+```
+
+The difference between Greek losses and US is instructive. Greece lost ships in the period September 1939 to October 1940. In a longer period of neutrality, the US lost just 16 ships, and, tellingly, only seven of those American ships were US flagged, seven were Panamanian, and two were under the Philippine colonial flag. According to the Greek language book by Christos Dounis, "En Kairo Polemou" (In Time of War), Greek ships scrupulously followed protocols for neutrals, \[\^5\] and while Greece was heroic in defending itself successfully against the Italian invaders, the country did its best not to provoke Germany and endured the loss of its ships and mariners. Yet this is not the full story, as Greek ships were basically seconded to the British war effort, as most Greek shipping was run out of London, and Greek shipping was an appendage of the British economy. \[\^6\] In spite of their neutral status, Greek ships were carrying supplies to Britain and the Germans therefore actively targeted them. In contrast, the US-flagged vessels were barred from carrying war materials to British or French ports under the Neutrality Acts of 1939, so non-US flagged ships had to do the carrying—and the dying.\[\^7\] This is precisely why the Panamanian flag was used, and tellingly, seven of the “US” losses were under the Panamanian flag. Adherence to the Neutrality Act helps to explain the huge difference in Greek versus American losses while both countries were still neutral. Like so many small countries, Greece had to pick its battles and fought heroically when attacked. Interestingly, one of the most comprehensive books about Greece in World War Two in the English language, “The First Victory,” makes no mention of the shipping losses during neutrality,\[\^8\] focusing instead of the land victory over Italy and the heroic fight against Germany and the resistance. Harlaftis, perhaps the foremost Greek nautical historian, provides a basic explanation of how Greek shipping could not remain neutral in spite of the Greek government’s dogged attempts to remain out of the carnage.
+
+```{r}
+
+AerialAtl <- Grships %>% 
+  filter(Event == "Aerial Bombing" & Location == "Atlantic")
+head(AerialAtl)
+```
+
+How were the Ships Sunk?
```

Comment:
This appears to be a sub heading. You should review markdown syntax: https://www.markdownguide.org/